swan-lake/development-tutorials/ballerina-persist/supported-data-stores.md

```diff
@@ -15,25 +15,66 @@ Following are the supported data stores.
 * In-memory
 * Relational databases (MySQL, MSSQL, PostgreSQL)
 * Google Sheets [Experimental]
+* Redis [Experimental]
 
-The data store is passed as a parameter to the `persist init` command. If you do not specify a data store, the in-memory data store is used by default.
+There are two ways to use `bal persist`.
+### Integrating to `bal build`
```

Comment:
```suggestion
### Integrate to `bal build`
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/8838/commits/63fc8376b7d8d4c2143b8d40ca938a46df3c9a0f